dev/vagrant/conf/consul/README.md

```diff
@@ -1,8 +1,5 @@
-# Overriding and appending consul configuration
+# Consul configuration
+You may add any hcl-files to this directory in order to change or override the configuration.
```

Comment:
Consul support hcl and json type files for config https://www.consul.io/docs/agent/options.html 

Or just remove this sentence, due to the next sentence cover idea about different file formats for configuration 
```suggestion
You may add any `.hcl` or `.json` -files to this directory in order to change or override the configuration.
```